esmvaltool/preprocessor/_io.py

```diff
@@ -42,6 +42,8 @@ def load_cubes(files, filename, constraints=None, callback=None):
     cubes = cubes.concatenate()
     for cube in cubes:
         cube.attributes['_filename'] = filename
+    if len(cubes) == 0:
```

Comment:
Replace by a more Pythonic
```
if not cubes:
```

Reply:
Yes, I have changed it also moved it before the for